### PR TITLE
Update API_RELEASE_NOTES.md

### PR DESCRIPTION
**Pull request by cfiorelli**
September release notes
- confirm no missing features or breaking changes
- suggest any edits for clarity 

thank you

### CODE REVIEW

**Comment by milescrawford** [comment]
Oh, you could also mention the new filtering and sorting features. Both search methods now support filtering on `publicationDateOrYear` and `minCitationCount` and the bulk search method now allows sorting the results by `publicationDate` or `citationCount`

File: API_RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,4 +1,43 @@
...
+
+### **New Features**
+* **Incremental Datasets**
+  * This feature allows you to update your datasets by downloading only the changes made since your last download. Benefits: Update datasets without downloading everything from scratch and save on download time and storage.
+  * [Relevant API Documentation ]([url](https://api.semanticscholar.org/api-docs/datasets#tag/Incremental-Updates))
+  * [Github code sample]([url](https://github.com/allenai/s2-folks/blob/main/examples/python/s2ag_datasets/incremental-updates.py))
+
+* **Paper Bulk Search**
+  * his feature is perfect for matching and filtering our paper data down to small sub-corpora. Our relevance search was limited in size, and our full corpus downloads involved sifting through many unrelated entries. Paper Bulk Search fits the use-case right in between!
+  * Benefits: Targeted Data Downloads: Isolate and download small sub-corpora of paper data. Allows matching and fetching up to 10,000,000 papers - far more than our relevance search method. Utilizes an optional simple boolean query against paper titles and abstracts. Sort order and page size are fixed for efficiency
```
Comment (rodneykinney) [inline API_RELEASE_NOTES.md:23]: "Allows matching and fetching up to 10,000,000 papers in batches of 1000"

"Results can be sorted by date, citation count, or paper ID"

File: API_RELEASE_NOTES.md [hunk 2]
```diff
@@ -7,8 +7,12 @@
   * In order to comply with legal we will be turning off API keys which have not accepted our updated terms of service by Monday October 16, 2023
   * Please review and accept the updated agreement [here]([url](https://www.semanticscholar.org/product/api/license))
 
+* **Reduction of Relevance Search maximum results from 10,000 to 1,000**
+  * On October 31st we reduce the maximum sum of offset and limit from 10,000 to 1,000 results.
+  * For needs which exceed this limitation please utilize our [open corpus]([url](https://api.semanticscholar.org/api-docs/datasets)).
```
Comment (rodneykinney) [inline API_RELEASE_NOTES.md:12]: Or the new bulk search endpoint!

**Review by rodneykinney — APPROVED** [review]

**Review by cfiorelli — COMMENTED** [review]
changes complete

**Comment by cfiorelli** [comment]
@rodneykinney Since miles is unavailable im proceeding without

**Pull request merged**